selfdrive/ui/qt/widgets/input.cc

```diff
@@ -220,6 +222,18 @@ ConfirmationDialog::ConfirmationDialog(const QString &prompt_text, const QString
     confirm_btn->setObjectName("confirm_btn");
     btn_layout->addWidget(confirm_btn);
     QObject::connect(confirm_btn, &QPushButton::clicked, this, &ConfirmationDialog::accept);
+    countdown = _count_down != 0 ? _count_down : 0;
```

Comment:
why not init it in ctor's member initializer lists?  `ConfirmationDialog(..., int count_down) : countdown(count_down), ... {`

Reply:
Fixed!